### DISCUSSION TITLE
chat corners glitch for a moment when pressing with mouse in multi

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#15629] [Q&A] [answered]
You can see it above the mouse when i left click (better seen in fullscreen)

https://user-images.githubusercontent.com/53081750/141751566-5af8ce5d-f7f5-4d0a-90bb-62cabb85a5d9.mp4

**Comment by smoogipoo** [comment]
This is going to be another case of nested masking, related to https://github.com/ppy/osu-framework/issues/2107.

Not sure how much of a priority this should be at this stage.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15636
